figures/palettes/oncoprint_color_palette.tsv

```diff
@@ -11,11 +11,13 @@ In_Frame_Ins	#FFE4E1
 Stop_Codon_Ins	#CC79A7
 Start_Codon_Del	#56B4E9
 Fusion	#7B68EE
-Multi_Hit	#00F021
+Multi_Hit	#CCCCCC
 Hom_Deletion	#313695
 Hem_Deletion	#abd9e9
```

Comment:
```suggestion
```
We are not actually using these

Reply:
We need to edit the script that generates them (`figures/scripts/color_palettes.R`), not this file directly. Editing this file directly caused problems (see the mentions of #1161 and #1189 in the original PR comment).

Reply:
gotcha!